terraform/modules/hub/prometheus.tf

```diff
@@ -393,13 +393,20 @@ resource "aws_dlm_lifecycle_policy" "prometheus" {
     resource_types = ["VOLUME"]
 
     schedule {
-      name = "24 hours weeks of hourly snapshots"
-      create_rule { interval = "cron(0 * * * ? *)" }
+      name = "24 hourly snapshots"
```

Comment:
I'm slightly confused by this description, can we make it more explicit to describe the interval and the retention window.